pandas/core/indexes/timedeltas.py

```diff
@@ -279,8 +280,25 @@ def _format_native_types(self, na_rep='NaT', date_format=None, **kwargs):
     # Wrapping TimedeltaArray
 
     @property
-    def _eadata(self):
-        return TimedeltaArray._simple_new(self._data, freq=self.freq)
+    def _data(self):
```

Comment:
can't these just be in the DatetimeIndexOpsMixin?

Reply:
We could, but would have to override for PeriodIndex.  Either way is fine by me.

Reply:
ok i c. either way is fine then.